template_studio.ini

```diff
@@ -0,0 +1,9 @@
+[general]
+component.name = HTML5Studio ;the application Name
+component.version = {{tagVersion}} ;similar to folder name
+
+[defaultKdp]
+widgets.studio.version = {{tagVersion}}
+widgets.studio.name = Video Studio V2
+widgets.studio.usage = html5studio
+widgets.studio.config = '{"version":"{{tagVersion}}", "name":"Video Studio V2", "tags":"studio_v2", "html5_version":"v2.63.3", "html5lib":"http://localhost/html5.kaltura/mwEmbed/mwEmbedLoader.php"}'
```

Comment:
@vladimir-poleh  why di we need it?
I didn't see use of it.

Reply:
It is used in \scripts\release.sh